sdl_android/src/main/java/com/smartdevicelink/transport/MultiplexBluetoothTransport.java

```diff
@@ -456,7 +453,7 @@ public void run() {
                         case STATE_LISTEN:
                         case STATE_CONNECTING:
                             // Situation normal. Start the connected thread.
-                        	getBluetoothSerialServerInstance().connected(socket, socket.getRemoteDevice());
+                        	connected(socket, socket.getRemoteDevice());
                             
                             break;
                         case STATE_NONE:
```

Comment:
For the case STATE_CONNECTED, the socket has already been verified to be non-null before entering the switch cases, so this null check statement can be removed.
`if(socket!=null){
                                	socket.close();
                                }
`